.golangci.yml

```diff
@@ -48,6 +49,17 @@ issues:
       text: "redefines-builtin-id"
 
 linters-settings:
+  depguard:
+    rules:
+      main:
+        deny:
+          - pkg: "github.com/containerd/containerd/errdefs"
+            desc: The containerd errdefs package was migrated to a separate module. Use github.com/containerd/errdefs instead.
+          - pkg: "github.com/containerd/containerd/log"
+            desc: The containerd log package was migrated to a separate module. Use github.com/containerd/log instead.
+          # TODO: enable once 1.6 migrated to github.com/containerd/platforms
+          # - pkg: "github.com/containerd/containerd/platforms"
+          #   desc: The containerd log package was migrated to a separate module. Use github.com/containerd/platforms instead.
```

Comment:
```suggestion
          #   desc: The containerd platforms package was migrated to a separate module. Use github.com/containerd/platforms instead.
```
small typo

Reply:
OH! Good catch; one copy-pasta too many. I think this one may already have slipped into the 1.7 release, so I'll do a follow-up there 😄

Reply:
Fixe the typo; PTAL 👍

Reply:
And opened https://github.com/containerd/containerd/pull/10371 for the 1.7 branch